pyproject.toml

```diff
@@ -3,12 +3,3 @@ first_party_detection = false
 
 [build-system]
 requires = ["setuptools", "wheel", "torch", "ninja"]
-
-first_party_detection = false
-
-target-version = ["py38"]
-
-excludes = [
```

Comment:
I think those excludes make sense

Reply:
Hmm probably but it does not belong to the `[build-system]` table.
After looking at the [PEP 518](https://peps.python.org/pep-0518) and [PEP 517](https://peps.python.org/pep-0517) specifications, it looks like only three keys are accepted in `[build-system]`: `requires`, `backend` and `backend-path`.